cli_tools/common/image/importer/inflater.go

```diff
@@ -60,6 +61,8 @@ const (
 	sigShadowInflaterErr  = "shadow err"
 )
 
+var allowedAPIFormatsEx = regexp.MustCompile("^(vpc)$")
```

Comment:
Interesting, I would have expected to see vmdk include here as well.

Reply:
I hadn't thought of that and may have had my blinders on here, @dntczdx should we still be testing vmdk formats as well?

Reply:
We still need to test VMDK - it's not even in GA yet.